src/components/Multisig/CreateMultisig.tsx

```diff
@@ -2,49 +2,65 @@
 // This software may be modified and distributed under the terms
```

Comment:
Threshold should be a number, available signatories and selected signatories do not work.